service/check_metadata.go

```diff
@@ -0,0 +1,77 @@
+package service
+import (
+	"github.com/yunify/qingstor-sdk-go/v3/request/errors"
+	"strings"
+)
+
+func IsMetaDataValid(XQSMetaData *map[string]string) error {
```

Comment:
Add this function in `utils` package instead.

Reply:
If Add this function into `utils` package，the `template/sub_service.tmpl` needed add `github.com/yunify/qingstor-sdk-go/utils`,and it will cause `imported and not used: "github.com/yunify/qingstor-sdk-go/utils"` in `service/bucket.go`. Maybe it's why `convert_types.go` is in `service`

Reply:
Add utils usage like `var _ fmt.State` to fix this.